view/frontend/web/template/payment/omise-cc-googlepay-form.html

```diff
@@ -1,22 +1,31 @@
-<div class="payment-method"
-    data-bind="css: {'_active': (getCode() == isChecked())}, visible: isActive()">
+<div class="payment-method" data-bind="css: {'_active': (getCode() == isChecked())}, visible: isActive()">
     <div class="payment-method-title field choice">
-        <input type="radio" name="payment[method]" class="radio"
-            data-bind="attr: {
-               'id': getCode()
-               },
-               value: getCode(),
-               checked: isChecked,
-               click: selectPaymentMethod,
-               visible: isRadioButtonVisible(),
-               enable: isActive()" />
-        <label data-bind="attr: {'for': getCode()}" class="label">
+        <input type="radio" name="payment[method]" class="radio" data-bind="attr: {
+                'id': getCode()
+            },
+            value: getCode(),
+            checked: isChecked,
+            click: selectPaymentMethod,
+            visible: isRadioButtonVisible(),
+            enable: isActive()"
+        />
+        <label data-bind="attr: {'for': getCode()}" class="omise-payment-method-title label">
             <span data-bind="text: getTitle()"></span>
+            <!-- ko if: getLogo() -->
+            <img alt="Gpay logo" data-bind="attr: { src: require.toUrl('Omise_Payment/'+logo.file), alt: logo.name },
```

Comment:
@aashishgurung @ajzkk Why is the alt text different from https://github.com/omise/omise-woocommerce/pull/368/files?

Reply:
@aashishgurung Can we use `Google Pay` or `GooglePay`? I will let you update and then I will follow yours.

Reply:
@danfowler  I will change it to `Google Pay logo`. But simply `Googlepay` does not meets the purpose of `alt` attribute.